<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="windowLayout" stretch="1,6">
      <item>
       <layout class="QHBoxLayout" name="controlsLayout">
        <item>
         <widget class="QGroupBox" name="dimensions_groupBox">
          <property name="title">
           <string>Graph Dimensions</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="dimensions_verticalLayout">
             <item>
              <widget class="QRadioButton" name="radioButton_3D">
               <property name="text">
                <string>3-D</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_2D">
               <property name="text">
                <string>2-D</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="geometry_groupBox">
          <property name="title">
           <string>Free Face Selection</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QVBoxLayout" name="geometry_verticalLayout">
             <item>
              <widget class="QRadioButton" name="radioButton_1Face">
               <property name="text">
                <string>1 Free Face</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_2Face">
               <property name="text">
                <string>2 Free Faces</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="stag_checkBox">
          <property name="text">
           <string>Apply Staggered Blast Pattern</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="presplitting_checkBox">
          <property name="text">
           <string>Design Presplit Holes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="design_pushButton">
          <property name="text">
           <string>Design</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="graphsLayout">
        <item>
         <widget class="GLViewWidget" name="openGLWidget"/>
        </item>
        <item>
         <widget class="PlotWidget" name="plotwidget" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLViewWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>pyqtgraph.opengl</header>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
